osychov/HW7/Task3.py

```diff
@@ -0,0 +1,10 @@
+def number_of_characters(string):
```

Comment:
good job
look at this implementation:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result

print(count_symbols_in_row("abracadabra"))

Reply:
Thanks for suggestion!